app/src/main.js

```diff
@@ -179,12 +193,18 @@ app.on('activate', function() {
 
 app.on('web-contents-created', function(event, contents) {
   contents.on('will-attach-webview', function(event, webPreferences, params) {
+    // Removing nodeIntegration breaks any plugin making use of native node bindings
+    // (those crash when used in Web Workers within Electron). As a workaround, we
+    // now use node's process.fork() to launch the worker as a node child process
+    // instead, and write the worker in such a way that it can handle being run
+    // in that manner in addition to the typical Web Worker path on the web.
+
     // Strip away preload scripts because they always have Node integration enabled
-    delete webPreferences.preload;
-    delete webPreferences.preloadURL;
+    // delete webPreferences.preload;
```

Comment:
This handler is designed to improve security for external content in [`webview`](https://electronjs.org/docs/api/webview-tag) tags. You shouldn't have to comment any of this out, unless you're using `webview` to do something in a plugin.